### PR TITLE
Trim down 'get back to safety' instructions

### PR DESCRIPTION
**Pull request by nathan-contino**
Closes https://github.com/raspberrypi/documentation/issues/3268

### CODE REVIEW

**Comment by nathan-contino** [comment]
Checking with @popcornmix, who should know for sure if this is the Correct Way.

**Comment by aallan** [comment]
Looking at #3268, looks like everyone is happy.

**Review by aallan — APPROVED** [review]
YOLO

File: documentation/asciidoc/computers/os/updating.adoc [hunk 1]
```diff
@@ -154,7 +154,7 @@ If you have done an `rpi-update` and things are not working as you wish, if your
 
 ----
 sudo apt-get update
-sudo apt install --reinstall libraspberrypi0 libraspberrypi-{bin,dev,doc} raspi-firmware
+sudo apt install --reinstall raspi-firmware
 ----
 
 NOTE: If you are still using Raspberry Pi OS Bullseye you will also need to reinstall the `raspberrypi-kernel` package.
```
Comment (lurch) [inline documentation/asciidoc/computers/os/updating.adoc:160]: Ahhh, ping @XECDesign and @popcornmix
With `libraspberrypi0 libraspberrypi-{bin,dev,doc}` being removed from the "how to undo rpi-update on Bookworm" instructions, do those packages now need to be explicitly added to this "how to undo rpi-update on Bullseye" note?

Comment (XECDesign) [reply documentation/asciidoc/computers/os/updating.adoc:160]: You can't really undo rpi-update in one line, so it depends on what the user actually wants to achieve. The best I can offer for bullseye is `sudo apt install --reinstall libraspberrypi0 libraspberrypi-{bin,dev,doc} raspberrypi-{kernel,bootloader}`. `raspi-firmware` shouldn't be there at all.

Comment (lurch) [reply documentation/asciidoc/computers/os/updating.adoc:160]: Yup, that's (pretty much) what it used to say before all the recent Bookworm tweaks: https://github.com/raspberrypi/documentation/commit/fef6e295f9ccb995e036fc9a23017558f94364c0

**Pull request merged**